Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/FileShareServiceCacheTest.cs

```diff
@@ -0,0 +1,176 @@
+﻿using FakeItEasy;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+using UKHO.FileShareClient.Models;
+using UKHO.MaritimeSafetyInformation.Common.Configuration;
+using UKHO.MaritimeSafetyInformation.Common.Helpers;
+using UKHO.MaritimeSafetyInformation.Common.Models.AzureTableEntities;
+using UKHO.MaritimeSafetyInformation.Common.Models.NoticesToMariners;
+using UKHO.MaritimeSafetyInformation.Web.Services;
+using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
+namespace UKHO.MaritimeSafetyInformation.Web.UnitTests.Services
+{
+    [TestFixture]
```

Comment:
I can't seem enough testing of InsertEntityAsync (e.g checking that the right data is written such as correct expiry time).  Can you please add?

Reply:
Because we are not actually insetting any data in table using unit test case, it will not be possible to check individual data in unit test case such as correct expiry time  
AzureTableStorageClient will take care of everything.
We were only checked flow off the program and faked AzureTableStorageClient.